src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -728,6 +727,10 @@ public int hashCode() {
      * @since 1.0.0
      */
     public static class StandardTypes {
+        private StandardTypes() {
+
+        }
```

Comment:
Since this is a breaking change, we might as well make the class `final`.

Reply:
Good point, a utility class should also be `final`.

I actually think that this class should be an `enum` instead, but then there is always the question about what to do with a non-standard type. A non-standard type would have a `StandardType` of `null` (or we could make another constant). But, you'd still have to keep type as `String` elsewhere, so you won't get any type-safety there.

At this point, we really would need a 2.0.0 release, c.f. the other pull requests I did to weaken the `TreeMap` type to `Map` which is also a breaking change.